lib/listening.js

```diff
@@ -1,14 +1,40 @@
 // Native
+const { existsSync, statSync } = require('fs')
 const path = require('path')
-const debounce = require('debounce')
 
 // Packages
+const ignoredDirectories = require('ignore-by-default').directories()
 const { write: copy } = require('clipboardy')
-const ip = require('ip')
-const chalk = require('chalk')
-const boxen = require('boxen')
+const getIgnoredFromGit = require('ignored')
 const { watch } = require('chokidar')
+const anymatch = require('anymatch')
+const debounce = require('debounce')
 const pkgUp = require('pkg-up')
+const chalk = require('chalk')
+const boxen = require('boxen')
+const ip = require('ip')
+
+const isDirSync = path => existsSync(path) && statSync(path).isDirectory()
+
+const getIgnoredFiles = ({ flags, gitignore, pkg }) => {
+  const set = new Set()
+  const addIgnore = value => set.add(value)
+
+  getIgnoredFromGit(gitignore).forEach(addIgnore)
+  if (flags.ignore) flags.ignore.forEach(addIgnore)
+  if (pkg.ignore) pkg.ignore.forEach(addIgnore)
+  ignoredDirectories.forEach(addIgnore)
+
+  const rawIgnored = Array.from(set)
+
+  const ignored = rawIgnored.reduce((acc, ignore) => {
```

Comment:
this is necessary for be sure ignore folder.

Explanation: https://github.com/es128/anymatch/issues/14#issuecomment-253338942